_data-prepper/pipelines/configuration/buffers/buffers.md

```diff
@@ -3,9 +3,17 @@ layout: default
 title: Buffers
 parent: Pipelines
 has_children: true
-nav_order: 20
+nav_order: 30
 ---
 
 # Buffers
 
-Buffers store data as it passes through the pipeline. If you implement a custom buffer, it can be memory based, which provides better performance, or disk based, which is larger in size.
\ No newline at end of file
+The `buffer` component acts as an intermediary layer between the `source` and `sink` components in a Data Prepper pipeline. It serves as a temporary storage for events, decoupling the `source` from the downstream processors and sinks. Buffers can be either in-memory or disk-based. 
+
+If not explicitly specified in the pipeline configuration, Data Prepper uses the default `bounded_blocking` buffer, which is in-memory queue bounded by the number of events it can hold. The `bounded_blocking` buffer is a convenient option when the event volume and processing rates are manageable within the available memory constraints. 
+
+Data Prepper also provides the flexibility to configure alternative buffer types based on your specific requirements. For example, <SME: What are the alternative types?>
```

Comment:
Those two are the only buffers currently supported. It may make sense to link to: https://opensearch.org/docs/latest/data-prepper/pipelines/configuration/buffers/buffers/

Reply:
This is the buffer.md page, lol. I'll remove the line for now, until we can point to somewhere that describes how to configure alternative types.